bindings/python/gen_docs.py

```diff
@@ -0,0 +1,111 @@
+from os.path import dirname
+from hashlib import md5
+import re
+
+base = dirname(dirname(dirname(__file__)))
+
+# This script runs with no CLI arguments
+# and scrapes documentation comments and c++ function signatures
+# from the manifold sources, in order to generate a c++ header file 
+# exposing the comments as string variables named by function names.
+# This allows python bindings to re-use the c++ comments directly.
+# Some snake-casing of params is applied for python use case.
+
+def snake_case(name):
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", name).lower()
+
+
+def param_snake_case(m):
+    return f":{snake_case(m[0][1:])}:"
+
+
+def method_key(name):
+    name = re.sub("\+", "_plus", name)
```

Comment:
We might need to add a substitution of `MeshGL` to `mesh`, since several docs refer to this data structure by its C++ name.